app/models/dish.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class Dish < ApplicationRecord
+  # scope by_name order(:type )
+  belongs_to :user
+  enum type: {
+    main_dish: main_dish = 'main',
```

Comment:
nope! You can't set variables here, set const. In rails const it is all uppercase.
main_dish: MAIN_DISH = 'main'